ncm-nss/src/test/resources/simple.pan

```diff
@@ -0,0 +1,15 @@
+object template simple;
+
+#function pkg_repl = { null; };
```

Comment:
remove these comments to start with :smile: 
and change https://github.com/quattor/configuration-modules-core/blob/master/ncm-nss/src/main/pan/components/nss/schema.pan#L11 in https://github.com/quattor/configuration-modules-core/blob/master/ncm-autofs/src/main/pan/components/autofs/schema.pan#L7
(and include any other types needed in the schema, but from quick glance, none are required)